Makefile

```diff
@@ -36,7 +38,7 @@ endif
 
 # The --path argument here restricts which BIP32 paths the app is allowed to derive.
 APP_LOAD_PARAMS = --path "44'/93'" --curve ed25519 $(COMMON_LOAD_PARAMS)
-ifeq ($(TARGET_NAME),TARGET_NANOS)
+ifeq ($(TARGET_NAME),$(filter $(TARGET_NAME),TARGET_NANOS TARGET_NANOS2))
 APP_LOAD_PARAMS += --appFlags 0x40
 else ifeq ($(TARGET_NAME),TARGET_NANOX)
 APP_LOAD_PARAMS += --appFlags 0x40
```

Comment:
Weird to see Stax having 0x200 to allow bluetooth and not Nanox, it doesn't seems right